src/components/v2/VaultItemUi/styles.ts

```diff
@@ -0,0 +1,112 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = () => {
+  const theme = useTheme();
+
+  return {
+    container: css`
+      width: 100%;
+    `,
+    header: css`
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      margin-bottom: ${theme.spacing(6)};
+    `,
+    title: css`
+      display: flex;
+      align-items: center;
+    `,
+    tokenIcon: css`
+      width: 20px;
+      height: 20px;
+      margin-right: ${theme.spacing(1)};
+    `,
+    tokenIconLarge: css`
+      width: 32px;
+      height: 32px;
```

Comment:
We should use `theme.iconSize` for these

Reply:
done